### PR TITLE
Project section

### PR DESCRIPTION
**Pull request by alejo2198**
complete project section with architecture. We'll do a lookover with the squad when all components are ready. Probably for cleanup, editing, and styling. also lmk if i did something wrong in type script 🤟

### CODE REVIEW

**Comment by moreiramj** [comment]
looks good and typescript seems good too. Biggest TS indicator that you did something wrong is problems output which your code has none. Couple nitpicky things
Can you move the content folder that holds project.json to the assets folder as its more of an asset than a component. Also remove project from the AppRoutes component it is still in there just commented out. And just maybe arrange the images in the assets folder used for the project into their own folder rather than raw in the assets. 

TLDR code good 👍 files in unholy places 🚮

**Comment by moreiramj** [comment]
looking 🔥 you made very good on the change request. Will happily merge
